internal/storage/bucket_handle_test.go

```diff
@@ -353,3 +353,21 @@ func (t *BucketHandleTest) TestListObjectMethodWithEmptyDelimiter() {
 	AssertEq(TestObjectGeneration, obj.Objects[0].Generation)
 	AssertEq(nil, obj.CollapsedRuns)
 }
+
+func (t *BucketHandleTest) TestListObjectMethodForMaxResult() {
```

Comment:
how many objects did you add in the fakeserver?

Reply:
four

Reply:
add a comment here that we have 4 objects in fakeserver

Reply:
fixed